packages/go/openapi/doc/openapi.json

```diff
@@ -12479,6 +12479,139 @@
         }
       }
     },
+    "/api/v2/attack-paths/finding-trends": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/header.prefer"
+        }
+      ],
+      "get": {
+        "operationId": "FindingTrendsForEnvironment",
+        "summary": "List finding trends",
+        "description": "Lists findings and their changes in between two dates for an environment",
+        "tags": [
+          "Attack Paths",
+          "Enterprise"
+        ],
+        "parameters": [
```

Comment:
`sort_by` isnt currently used by the UI, so Im not requesting it get added for that but Im wondering if it would be breaking the API pattern to not include it here?

Reply:
no, we don't currently support any filtering or sorting logic on this endpoint aside from the requested env ID's